db/trie/mptrie/branchnode.go

```diff
@@ -69,6 +72,9 @@ func newRootBranchNode(cli client, children map[byte]node, indices *SortedList,
 			}
 		}
 	}
+	if err := logNode(_nodeTypeBranch, _actionTypeNew, bnode, cli); err != nil {
```

Comment:
why is `logging` used at the end of function instead of at the head?

Reply:
Wait for the  `bnode` Initialized